app/templates/main/error.hbs

```diff
@@ -1,7 +0,0 @@
-{{#if slug}}
```

Comment:
@drogus This seems like a reasonable change given the removal of the `main` route, but I don't want to have made our UX worse with this PR. We didn't have test coverage around this, so not sure if we really care about this or not. Thoughts?